packages/types/config/build.d.ts

```diff
@@ -157,6 +157,7 @@ export interface NuxtOptionsBuild {
   html?: { minify: HtmlMinifierOptions }
   indicator?: boolean
   loaders?: NuxtOptionsLoaders
+  loadingScreen?: false | {}
```

Comment:
```suggestion
  loadingScreen?: boolean| any
```

(more precisely https://github.com/nuxt/screens/blob/7495e39d27c42696b5436088a3904fd8e3871431/packages/loading/lib/module.js#L12)

Reply:
sure, I wasn't sure anyway if these were the only allowed values :+1: